docs/develop/api-reference/auth-tokens.md

```diff
@@ -161,14 +161,15 @@ const TopicsPermissions = {
 Generates a new disposable Momento auth token with the specified permissions and expiry.
 
 Disposable tokens differ from the usual Momento auth token in several key ways:
-  - They cannot be generated in the console, they can only be generated programatically. 
+  - They cannot be generated in the console, they can only be generated programatically. The token that's used for the `generateDisposableToken` API call must be a token with Super User scope generated via the Momento console.
+  - They must expire within one hour.
   - They cannot be refreshed and thus do not come with a refresh token. 
-  - Permissions are specified using DisposableTokenScope. Disposable tokens can permit access to specific cache items by specifying the key or key-prefix. This can be for all caches or just the caches you specify. Permissions cannot be applied to Topics.
+  - Permissions are specified using the usual TokenScope object or using DisposableTokenScope object, which allows you to permit access to specific cache items by specifying a key or key-prefix. 
 
 | Name            | Type                      | Description                                                                                                                                                                             |
 | --------------- |---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | scope           | [DisposableTokenScope](#disposabletokenscope-objects) | The permissions to grant to the new disposable token. Pre-built DisposableTokenScope objects are provided by the SDKs.                                                                                       |
-| expiresIn       | Number&nbsp;&nbsp;\|&nbsp;&nbsp;ExpiresIn&nbsp;object | The number of seconds until the token expires or an ExpiresIn object representing a duration by calling the `ExpiresIn.never()`, `ExpiresIn.minutes()`, or `ExpiresIn.hours()` methods. |
+| expiresIn       | Number&nbsp;&nbsp;\|&nbsp;&nbsp;ExpiresIn&nbsp;object | The number of seconds until the token expires or an ExpiresIn object representing a duration by calling the `ExpiresIn.minutes()` or `ExpiresIn.hours()` methods. |
```

Comment:
Sorry, I forgot to mention in my previous review that max expiry time for a disposable token is 1 hour. All other changes look good.